release-notes/2.0/releases.json

```diff
@@ -0,0 +1,639 @@
+                        {   "name":"runtime-mac-x64.pkg", 
+                            "url":"https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-osx-x64.pkg", 
+                            "hash": "fd1f5e4e6ed08f47e87b907d1c28adcfb983b53822614a01fe23a8d1d81862e2b75b7ebfc4e28770f7486080d72e81179ebb05b569be5d7c2410bf52bf1270c2"},
+                        {   "name":"runtime-win-x86", 
+                            "url":"https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-win-x86.zip", 
+                            "hash": "e73a78ee66f872678594d01720849a3b95a4911ded6700a55b2dbc35fa54a42aeadeeb3de0706a1b96c7b2ed383bc431b8c6e36ce8082bb673132b3d80c7d441"},
+                        {   "name":"runtime-win-x64", 
+                            "url":"https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-win-x64.zip", 
+                            "hash": "4e03072925ca65f350112916eac8f442ee86f84de5fd8643d6db9eb96e2175351fdee0fb8ebed6a508b5ce2a3f73bdd4b7a084adc0c765c5249c019f601179f4"},
+                        {   "name":"runtime-win-x86.exe", 
+                            "url":"https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-win-x86.exe", 
+                            "hash": "159faebadf18cd2e06b3ca1b71f28a65ad6c3a4be78a090fb1f0947951d392d5bbd9bca9055610658c4e16dab66bc1e13200ac869b88725f6e4ba91de2f62822"},
+                        {   "name":"runtime-win-x64.exe", 
+                            "url":"https://download.microsoft.com/download/3/a/3/3a3bda26-560d-4d8e-922e-6f6bc4553a84/dotnet-runtime-2.0.9-win-x64.exe", 
+                            "hash": "9d8ba6ae17a5519a8a4ed043919114d98cf36595c5c3fc82ed699c225b56e07d782680e4a57b1a83f4c9073b75045c89770ee4b0d997a8c19776d1e946bde478"}
+                    ]
+                },
+                "sdk": {
+                    "version": "2.1.202",
+                    "version-display":"2.1.202",
+                    "vs-version":"15.7",
+                    "lts": false,
+                    "eol": false,
+                    "csharp-language": "7.2",
+                    "checksums": "https://dotnetcli.blob.core.windows.net/dotnet/checksums/2.1.202-sdk-sha.txt",                
+                    "files":[
+                        {   "name":"sdk-linux-x64", 
+                            "url":"https://download.microsoft.com/download/f/c/1/fc16c864-b374-4668-83a2-f9f880928b2d/dotnet-sdk-2.1.202-linux-x64.tar.gz", 
+                            "hash": "c1b07ce8849619ca505aafd2983bcdd7141536ccae243d4249b0c9665daf107e03a696ad5f1d95560142cd841a0888bbf5f1a8ff77d3bdc3696b5873481f0998"},
+                        {   "name":"sdk-mac-x64", 
```

Comment:
I have been treating it as an unversioned identifier rather than a filename. Will need to think through changing it to mean 'filename'.

Reply:
The part that is confusing to me is that it is being treated as an intelligent key/identifier yet the value is not unique enough to distinguish the difference between the various files.  By that I mean `"name": "sdk-win-x64"` doesn't contain enough information to distinguish between the zip and exe.  This could lead to issues with consuming code.  IMO filename would be a better choice here.

Reply:
Updated `name` values to be what we chatted about in IM. eg filename=dotnet-runtime-2.1.5-linux-arm.tar.gz would yield name=dotnet-runtime-linux-arm.tar.gz